src/main/kotlin/app/revanced/patches/youtube/layout/playerbuttonoverlay/patch/PlayerButtonOverlayPatch.kt

```diff
@@ -21,13 +21,14 @@ import org.w3c.dom.Element
 @PlayerButtonOverlayCompatibility
 @Version("0.0.1")
 class PlayerButtonOverlayPatch : ResourcePatch {
-    private val resourceFilePath = "res/drawable/player_button_circle_background.xml"
-
-    // the attributes to change the value of
-    private val replacements = arrayOf(
-        "color"
-    )
-
+    private companion object {
+        const val resourceFilePath = "res/drawable/player_button_circle_background.xml"
```

Comment:
Nitpicking here, but we want to remain more consistent as the project grows, constants are written LIKE_THIS.

Reply:
To help with this in the future, you could look into a Kotlin linter and then invoke that in a CI job.
That way, a PR cannot be merged unless the code is following the guidelines.
For example https://pinterest.github.io/ktlint/